src/main/java/de/tum/in/www1/artemis/web/rest/LectureResource.java

```diff
@@ -70,50 +68,50 @@ public LectureResource(LectureRepository lectureRepository, LectureService lectu
     }
 
     /**
-     * POST /lectures : Create a new lecture.
+     * POST lectures : Create a new lecture.
      *
      * @param lecture the lecture to create
      * @return the ResponseEntity with status 201 (Created) and with body the new lecture, or with status 400 (Bad Request) if the lecture has already an ID
      * @throws URISyntaxException if the Location URI syntax is incorrect
      */
-    @PostMapping("/lectures")
+    @PostMapping("lectures")// TODO: should be /courses/{courseId}/lectures
     @PreAuthorize("hasRole('EDITOR')")
     public ResponseEntity<Lecture> createLecture(@RequestBody Lecture lecture) throws URISyntaxException {
         log.debug("REST request to save Lecture : {}", lecture);
         if (lecture.getId() != null) {
             throw new BadRequestAlertException("A new lecture cannot already have an ID", ENTITY_NAME, "idexists");
         }
         User user = userRepository.getUserWithGroupsAndAuthorities();
-        if (!authCheckService.isAtLeastEditorInCourse(lecture.getCourse(), user)) {
-            return forbidden();
-        }
+        Long courseId = lecture.getCourse().getId(); // use the courseId in path instead after refactoring, then check if id matches in body
```

Comment:
I think you should add a check whether the course object in the lecture exists otherwise it may throw a null pointer exception